src/writer.js

```diff
@@ -0,0 +1,377 @@
+      elem.setAttribute(keysValues[i-1], keysValues[i]);
+      i += 2;
+    }
+  }
+  return elem;
+};
+
+var classArrLength = 0;
+export function addCards (num) {
+  let classArray = ['card1', 'card1', 'card2', 'card2', 'card3', 'card3', 'card4', 'card4', 'card5', 'card5', 'card6', 'card6', 'card7', 'card7', 'card8', 'card8', 'card9', 'card9', 'card10', 'card10', 'card11', 'card11', 'card12', 'card12', 'card13', 'card13', 'card14', 'card14', 'card15', 'card15', 'card16', 'card16', 'card17', 'card17', 'card18', 'card18'];
+
+  function getRndInt(min, max) {
+    return Math.floor(Math.random() * (max - min) ) + min;
+  }
+
+  switch (document.getElementsByClassName('fieldSelector')[0].value) {
+    case '44':
+      classArray.length = 16;
+      classArrLength = classArray.length;
+      break;
+    case '54':
+      classArray.length = 20;
+      classArrLength = classArray.length;
+      break;
+    case '66':
+      classArray.length = 36;
+      classArrLength = classArray.length;
+      break;
+  }
+
+  for(let i = 0; i < classArrLength; i++){
+    let random = getRndInt(0, classArray.length);
+    document.getElementsByClassName('gameField')[0].append(createHtml('div', `card ${classArray[random]} cardBack`));
+    classArray.splice(random, 1);
+  }
+}
+
+export function stopTurns () {
+  document.getElementsByClassName('turnsCounter')[0].value = '0';
+}
+
+//toggling card face up/down
+export function toggleCard (...cards) {
+  for(let card of cards){
+    card.classList.toggle('cardBack');
+  }
+}
+
+export function showMsg (msg) {
+  document.getElementsByClassName('gameField')[0].prepend(createHtml('div', 'msgDiv'));
+  document.getElementsByClassName('msgDiv')[0].prepend(createHtml('p', '', msg));
+};
+
+var emptyCardNumber = 0;
+export function compareCards (card1, card2) {
+  if (card1.classList[1] == card2.classList[1]) {
+    setTimeout(() => {card1.classList.add('invisible', 'noclick');}, 300);
+    setTimeout(() => {card2.classList.add('invisible', 'noclick');}, 300);
+    emptyCardNumber += 2;
+  } else {
+    card1.classList.remove('noclick');
+    card2.classList.remove('noclick');
+    setTimeout(toggleCard, 400, card1, card2);
+  }
+}
+
+export function changeFieldSize () {
+  let elems, field, card;
+  isLoadedGame = false;
+
+  document.getElementsByClassName('gameField')[0].remove();
+
+  elems = document.getElementsByClassName('fieldSelector');
+  Array.from(elems).forEach((el) => {
+    field = document.createElement('div');
+    field.className = 'gameField flex';
+    document.getElementsByClassName('timeNturnsBlock')[0].after(field);
+
+    clearStopwatch();
+    clearTimeout(clocktimer);
+    init = 0;
+    stopTurns();
+    isFirstClick = true;
+    emptyCardNumber = 0;
+
+    showMsg('<b>Shuffling...</b>');
+    setTimeout(function () {
+      document.getElementsByClassName('msgDiv')[0].remove();
+    }, 1000);
+
+    switch (el.value) {
+      case '44':
+        field.classList.add('gameField--4x4');
```

Comment:
please do not use magic words. The problem of strings is when you are using them you can type it incorrecly and will not notice that. Also I cannot understand what values can be used (in case on enum analogue). 

to avoid that make constants.js file when you specify all strings & numbers :
```
/* constants.js*/
const selectors = {
  field.gameField54 : 'gameField--5x4',
 ....
}

...

/* your file */
field.classList.add(selectors.gameField54);
```

Reply:
Done